sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -618,4 +620,43 @@ export class ApplicationStudentsController extends BaseController {
       }
     }
   }
+
+  /**
+   * Get application to request appeal.
+   ** Application eligible to be requested for
```

Comment:
Are the special comments with "*" ignored when the swagger is generated?
If not, I would suggest to avoid them 😉

Reply:
The special comments are fine if the swagger can deal with them, as per your screenshot, it does nicely 😉 
I was afraid that the additional "*" would be in the middle of the comments.
Let's keep using them as much as needed 👍